currencyii/src/main/java/nl/tudelft/trustchain/currencyii/coin/WalletManager.kt

```diff
@@ -112,9 +112,13 @@ class WalletManager(
         })
 
         Log.i("Coin", "Coin: starting the setup of kit.")
-        kit.setBlockingStartup(false)
-            .startAsync()
-            .awaitRunning()
+        try {
+            kit.setBlockingStartup(false)
+                .startAsync()
+                .awaitRunning()
+        } catch (t: Throwable) {
```

Comment:
Why do we want to catch the exception here?

Reply:
Before the stability fixes, I used this to see what caused the crashes. I can remove the try-catch, need to make sure they are caught somewhere else. I will see if I can remove it and catch it where they are called from the UI, so we can show the user that something went wrong.

Reply:
Yes better to not catch the error here

Reply:
Fixed: https://github.com/InvictusRMC/trustchain-superapp/pull/14/commits/f47db8d093de2d3a8e5e2532b68a870dd47f96dd